elasticdl/python/worker/worker.py

```diff
@@ -469,6 +504,22 @@ def run(self):
             mode = Mode.EVALUATION
         else:
             mode = Mode.TRAINING
+
+        # need to get model from PS, i.e. `train_with_local_model=False`
+        # at following situations:
```

Comment:
Make it clear:
The worker needs to get model from PS if `train_with_local_model=False`. 
This happens when:

Reply:
Done.